### ISSUE TITLE
Question about the masked memory

### ISSUE THREAD

**Issue by tjdhg456** [scok30/MAE-CIL#2]
Hello! thank you for your work.

I have one more question about masked examples.
It seems your mae_memory examplar folder contains original 8000 images rather than the MAE decoded images.
In the code it also save the original images.

Is it right? it will be updated ?

Thank you.

**Comment by scok30** [comment]
Regarding this matter, in fact, you will find that longer training times are required when conducting experiments. At the end of each task, we need to ensure that MAE can reconstruct the image of the current task. This is, in fact, an overfitting issue on the training set. Previous experiments related to MAE have often required a longer number of pre-training epochs, and the version we have currently uploaded also takes into consideration reducing the training cost for other researchers and enhancing reproducibility. Once our future work reaches a certain point, we will upload the complete code for using MAE to reconstruct the replay buffer.

**Comment by tjdhg456** [comment]
Thank you for your reply. I understand your reply.
Training MAE with longer time means 400 epochs as mentioned in paper?
Or it will require more epoch? (because found that uncommented lines about epochs=1600)
This is my last question. Thank you for your kind reply.

**Comment by scok30** [comment]
Our experience is that pre-training typically requires approximately 400-800 epochs. You can refer to the explanation provided in the article 'SupMAE: Supervised Masked Autoencoders Are Efficient Vision Learners' for more details.
